### PR TITLE
New Pok3r Based Keymap

### PR DESCRIPTION
**Pull request by s1carii**
Got this working over the weekend, compiles without error and follows most Pok3r shortcut/layer conventions. 

Also disabled LEDs.

### CODE REVIEW

File: keyboards/iris/keymaps/s1carii/keymap.c [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#include "iris.h"
```
Comment (drashna) [inline keyboards/iris/keymaps/s1carii/keymap.c]: Could you replace the includes with just `#include QMK_KEYBOARD_H`, as that's all that's needed here.

File: keyboards/iris/keymaps/s1carii/rules.mk [hunk 2]
```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = no
+BACKLIGHT_ENABLE = no
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/iris/keymaps/s1carii/rules.mk]: This block isn't needed

Comment (s1carii) [reply keyboards/iris/keymaps/s1carii/rules.mk]: The make fails without this block so I've replaced that for now. Unless I've misinterpreted what you mean here, is there another place that's referenced that I have to change as well? 

Thanks for the help here!

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**